layers/+lang/python/funcs.el

```diff
@@ -35,6 +35,16 @@
       (setq python-shell-interpreter-args "-i")
       (setq python-shell-interpreter "python"))))
 
+(defun spacemacs/python-setup-checkers (&rest args)
+  (progn
+    (if (spacemacs/pyenv-executable-find "pylint")
+        (flycheck-set-checker-executable "python-pylint"
+                                         (spacemacs/pyenv-executable-find "pylint")))
+    (if (spacemacs/pyenv-executable-find "flake8")
+        (flycheck-set-checker-executable "python-flake8"
+                                         (spacemacs/pyenv-executable-find "flake8")))
+    ))
```

Comment:
1. Please use a loop (`dolist` will probably suffice) to avoid code duplication.
2. Use `when` instead of `if` when you don't have an `else` clause.
3. (optional) use `let` to bind the result of `spacemacs/pyenv-executable-find` instead of calling it twice. What I mean:
```elisp
(let ((path (spacemacs/pyenv- ...)))
  (when path
    (flycheck-set- ... path)))
```